apps/api/src/server/repositories/document-activity-log.repository.js

```diff
@@ -2,7 +2,8 @@ import { databaseConnector } from '../utils/database-connector.js';
 
 /**
  *
- * @param {import('@pins/applications.api').Schema.DocumentActivityLog} documentLog
+ * @typedef {{documentGuid: string, version: number, user: string, status: string}} CreateLogInput
```

Comment:
could we use one of Prisma's predefined types here?  in my refactor of the types that I am working on, I changed this line to:
@param {import('@prisma/client').Prisma.DocumentActivityLogUncheckedCreateInput} documentLog
(this fixes the type redline errors in the calls too)